README.md

```diff
@@ -41,13 +41,24 @@ require('http2').createServer(options, function(request, response) {
 ### Using as a client ###
 
 ```javascript
-process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
-
 require('http2').get('https://localhost:8080/', function(response) {
   response.pipe(process.stdout);
 });
 ```
 
+Self-signed certificates must be specified for each request to avoid
```

Comment:
Seems reasonable. I imagine self-signed certs would be more useful for internal infrastructure, but folks who do that already need to create their own `https.Agent`.